keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/ArtifactVersioned.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.api
+
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+
+interface ArtifactVersioned {
```

Comment:
Perhaps a better name for this would be something like `ArtifactVersionProvider` or `ArtifactVersionAware`?